tests/test_pex.py

```diff
@@ -19,6 +19,24 @@ def test_pex_uncaught_exceptions():
   assert rc == 1
 
 
+def test_excepthook_honored():
+  body = textwrap.dedent("""
+  import sys
+
+  def excepthook(ex_type, ex, tb):
+    print('Custom hook called with: {}'.format(ex))
```

Comment:
I learned that '{}'.format() is not supported in Python 2.6.  '{0}'.format() instead.